### ISSUE TITLE
stack metod defined twice

### ISSUE THREAD

**Issue by Cor3Down** [tensorly/tensorly#133]
The ' **stack** ' static method **defined twice** in tensorly/backend/pytorch_backend.py
`    @staticmethod
      def stack(arrays, axis=0):
           return torch.stack(arrays, dim=axis)
    `
